### DISCUSSION TITLE
Setting theme based on an environment variable

### DISCUSSION THREAD

**Discussion by JeffofBread** [davatorium/rofi#2061] [Q&A]
Is there a way I could dynamically switch between themes natively in rofi? I could script it of course with a find and replace, but that seems a little cumbersome. I have tried things like ```@theme env(VAR, fallback_theme)``` or ```@theme $VAR```, but neither have worked. Both just throw config errors. Is there a way I am missing to do this natively in rofi?

**Comment by DaveDavenport** [comment]
No, I don't think there is an easy way to do this atm..  You can change the theme using environment variables (see website) but not load a completely different theme.

On the commandline it is doable...  `rofi -theme $THEME`
